src/sys/unix/net.rs

```diff
@@ -1,8 +1,8 @@
-use std::io;
-use std::mem::size_of_val;
+#[cfg(any(feature = "tcp", feature = "udp"))]
```

Comment:
Should this entire module be conditionally used? By adding `cfg` above `mod net` in `mod.rs`?